agent/pkg/chassis/registry/registry.go

```diff
@@ -96,14 +96,16 @@ func (esd *EdgeServiceDiscovery) FindMicroServiceInstances(consumerID, microServ
 	}
 
 	// gen MicroServiceInstances
+	// TODO(Poorunga) needs optimization
```

Comment:
If you need a todo here, you should make it clear, what do you want to  optimizaton? Or we just delete the comment.

Reply:
OK, delete it.